.pylintrc

```diff
@@ -92,6 +93,10 @@ disable=
     # W9018, # "%s" differing in parameter type documentation
     # W9019, # "%s" useless ignored parameter documentation
     # W9020, # "%s" useless ignored parameter type documentation
+    C0103, # Class constant name "%s" doesn't conform to UPPER_CASE naming style ('([^\\W\\da-z][^\\Wa-z]*|__.*__)$' pattern) (invalid-name)
```

Comment:
Can we get some short justification in the PR description for disabling these?

Reply:
Thanks! Added.